js/setup.js

```diff
@@ -0,0 +1,53 @@
+'use strict';
+
+const NAMES = [`Иван`, `Хуан Себастьян`, `Мария`, `Кристоф`, `Виктор`, `Юлия`, `Люпита`, `Вашингтон`];
+const SURNAMES = [`да Марья`, `Верон`, `Мирабелла`, `Вальц`, `Онопко`, `Топольницкая`, `Нионго`, `Ирвинг`];
+const COAT_COLORS = [`rgb(101, 137, 164)`, `rgb(241, 43, 107)`, `rgb(146, 100, 161)`, `rgb(56, 159, 117)`, `rgb(215, 210, 55)`, `rgb(0, 0, 0)`];
+const EYES_COLORS = [`black`, `red`, `blue`, `yellow`, `green`];
+
+const setup = document.querySelector(`.setup`);
+setup.classList.remove(`hidden`);
+const similarWizardTemplate = document.querySelector(`#similar-wizard-template`).content.querySelector(`.setup-similar-item`);
+const charactersList = document.querySelector(`.setup-similar-list`);
+const similarCharacters = document.querySelector(`.setup-similar`);
+similarCharacters.classList.remove(`hidden`);
+
+//  Функция генерации случайных данных
+const createCharactersData = function (names, surnames, coatColors, eyesColors) {
+  const charactersData = [];
+  for (let i = 0; i < 4; i++) {
+    charactersData[i] = {
+      name: `${names[Math.floor(Math.random() * names.length)]} ${surnames[Math.floor(Math.random() * surnames.length)]}`,
+      coatColor: coatColors[Math.floor(Math.random() * coatColors.length)],
+      eyesColor: eyesColors[Math.floor(Math.random() * eyesColors.length)]
+    };
+  }
+
+  return charactersData;
+};
+
+//  Создание массива со случайными данными
+const charactersData = createCharactersData(NAMES, SURNAMES, COAT_COLORS, EYES_COLORS);
+
```

Comment:
```suggestion
//  Создание массива со случайными данными
const charactersData = createCharactersData();

```
Здесь соответсвенно параметры не передаем